core/src/components/select/select.md.vars.scss

```
@@ -19,6 +19,9 @@ $select-md-padding-start:       $item-md-padding-start !default;
 /// @prop - Color of the select icon
 $select-md-icon-color:          $text-color-step-600 !default;
 
+/// @prop - Size of the select icon
+$select-md-icon-size: 13px !default;
```

```suggestion
$select-md-icon-size:           13px !default;
```

Same as ios, aligning the variables. May need to adjust all to align with `$select-md-floating-label-padding`